packages/nitro-protocol/src/contract/state.ts

```diff
@@ -12,16 +12,14 @@ export interface State {
   appDefinition: string;
   appData: string;
 }
-
-export function getFixedPart(
-  state: State,
-): {
+export interface FixedPart {
   chainId: Uint256;
   participants: Address[];
   channelNonce: Uint256;
   appDefinition: Address;
   challengeDuration: Uint48;
-} {
```

Comment:
Could we try to organize types of this nature into logical buckets? Perhaps `FixedPart` should be within `@statechannels/nitro-protocol` because it is a contract-level structure that happens to bleed a little bit into the client code.